src/polyphy/utils/metadata.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+
+def get_metadata(ppConfig, ppInternalData):
+    metadata = {
+        "_dataset": ppConfig.ppData.input_file,
```

Comment:
please remove underscore from the beginning of the string

Reply:
Done